torch/ao/quantization/experimental/APoT_tensor.py

```diff
@@ -2,11 +2,11 @@
 from torch.ao.quantization.experimental.quantizer import APoTQuantizer
 
 # class to store APoT quantized tensor
-class TensorAPoT(torch.Tensor):
+class TensorAPoT():
     quantizer: APoTQuantizer
 
     def __init__(self, quantizer):
-        raise NotImplementedError
+        self.quantizer = quantizer
 
-    def int_repr(self):
-        raise NotImplementedError
+    def int_repr(self, tensor: torch.Tensor):
+        return self.quantizer.quantize_APoT(tensor)
```

Comment:
see above comment and https://pytorch.org/docs/stable/generated/torch.Tensor.int_repr.html. 
`int_repr` is a getter function. it shouldn't be quantizing anything